assets/js/content.js

```diff
@@ -156,9 +156,11 @@ async function showDonwvotes(art_id,json){
     }                                   
   }
   ////console.log('postId ',post_id," downvotes", downvotes);
-  if(downvotes!== null){
-    $("#"+art_id+" .post-vote").append(`<span class="post-vote__text downvote">${downvotes}</span>`);
```

Comment:
no mention of this line in the new code. This line is for mobile, the next is for desktop (the use different DOM and classes). Without this, downvotes will show only on desktop not on mobile.